src/book/02-system/03-tools/02-nubook.mdx

```diff
@@ -28,7 +28,13 @@ You'll need [Git](https://git-scm.com/), [Node.js](https://nodejs.org/en/) and [
 
 2. Install the latest version of Node.js by following the instructions [here](https://github.com/nodesource/distributions/blob/master/README.md#installation-instructions) (pick the latest version).
```

Comment:
```suggestion
2. Install Node.js version 14 by following the instructions [here](https://github.com/nodesource/distributions/blob/master/README.md#installation-instructions) and running the commands under "Node.js v14.x".
```

since the latest doesn't seem to work

Reply:
Yes. Was thinking to put that but got slack, absolutely should.